ajax/libs/FormChimp/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "FormChimp",
+  "filename": "jquery.formchimp.min.js",
+  "title": "FormChimp",
+  "description": "A customizable MailChimp ajax plugin for jQuery",
+  "version": "1.2.1",
+  "author": {
+    "name": "Fabio Quarantini",
+    "url": "http://www.fabioquarantini.com",
+    "twitter": "@febba"
+  },
+  "homepage": "http://www.fabioquarantini.com/formchimp/",
+  "keywords": [
+    "FormChimp",
+    "MailChimp",
+    "jQuery plugin",
+    "ajax"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/fabioquarantini/formchimp.git"
+  },
+  "licenses":	{
+    "type": "MIT",
+    "url": "http://opensource.org/licenses/MIT"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/fabioquarantini/formchimp.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.formchimp.min.js",
+          "jquery.formchimp.js"
```

Comment:
We usually use [node-glob](https://github.com/isaacs/node-glob#glob-primer) which is the pattern mechanism to get the files.
Let's revise from A to B.

- A
```js
          "jquery.formchimp.min.js",
          "jquery.formchimp.js"
```

- B
```js
          "jquery.formchimp*.+(js|css|map)"
```

Thank you.